platforms/hyperledger-fabric/configuration/deploy-network.yaml

```diff
@@ -111,6 +111,7 @@
         org_name: "{{ item.name }}"
         services: "{{ item.services }}"
         subject: "{{ item.subject }}"
+        external_url_suffix: "{{ item.external_url_suffix }}"
```

Comment:
This change is not needed as in the next file you are using item.external_url_suffix  again